server/src/concepts/offer/useCases/getOffers.js

```diff
@@ -0,0 +1,7 @@
+// not sure if i need to create command for getOffers
```

Comment:
for getting items from DB we are using queries not commands, commands are for creating, updating, deleting items